src/components/form-feedback/form-feedback.js

```diff
@@ -0,0 +1,195 @@
+
+    }
+
+    onBlurEmailHandler = (e) => {
+        const email = e.target.value
+        const isValidEmail = this.regExpEmail.test(email)
+        if(!isValidEmail){
+            this.labelEmail.classList.add('invalid')
+            this.inputEmail.classList.add('invalid')
+            this.labelEmail.classList.remove('valid')
+            this.inputEmail.classList.remove('valid')
+            this.pEmailError.textContent = 'Неверный e-mail'
+            this.formFieldsState.email = false
+            this.formValidation()
+
+        }else{
+            this.labelEmail.classList.remove('invalid')
+            this.inputEmail.classList.remove('invalid')
+            this.labelEmail.classList.add('valid')
+            this.inputEmail.classList.add('valid')
+            this.formFieldsState.email = true
+            this.formValidation()
+        }
+    }
+
+    onFocusEmailHandler = () => {
+        this.pEmailError.textContent = ''
+    }
+
+    onBlurMessageHandler = (e) => {
+        const message = e.target.value.trim()
+
+        if(message === ''){
+
+            this.labelMessage.classList.add('invalid')
+            this.inputMessage.classList.add('invalid')
+            this.labelMessage.classList.remove('valid')
+            this.inputMessage.classList.remove('valid')
+            this.pMessageError.textContent = 'Сообщение не может быть пустым'
+            this.formFieldsState.message = false
+            this.formValidation()
+
+        }else if (message.length >= 1000){
+
+            this.labelMessage.classList.add('invalid')
+            this.inputMessage.classList.add('invalid')
+            this.labelMessage.classList.remove('valid')
+            this.inputMessage.classList.remove('valid')
+            this.pMessageError.textContent = 'Сообщение не может содержать больше 1000 символов'
+            this.formFieldsState.message = false
+            this.formValidation()
+
+        }else {
+            this.labelMessage.classList.remove('invalid')
+            this.inputMessage.classList.remove('invalid')
+            this.labelMessage.classList.add('valid')
+            this.inputMessage.classList.add('valid')
+            this.formFieldsState.message = true
+            this.formValidation()
+        }
+    }
+
+    onFocusMessageHandler = () =>{
+        this.pMessageError.textContent = ''
+    }
+
+    onChangeFileHandler = (e) =>{
+        const files = [...e.target.files]
+        this.labelFile.classList.remove('invalid')
+        this.wrapperFiles.innerHTML = ''
+
+        if(files.length > 2){
+            this.labelFile.classList.add('invalid')
+            this.pFileError.textContent = 'Нельзя выбрать больше 2-х файлов'
+            this.formFieldsState.files = false
+            this.formValidation()
+        }else if(files.length <= 2){
+            this.labelFile.classList.remove('invalid')
+            this.pFileError.textContent = ''
+            this.buildElement(files)
+        }
```

Comment:
В данном случае можно обойтись без дополнительной проверки
```suggestion
        }else {
            this.labelFile.classList.remove('invalid')
            this.pFileError.textContent = ''
            this.buildElement(files)
        }
```